docs/guides/Composability/vtex-composable-and-complete.md

```diff
@@ -5,21 +5,25 @@ hidden: true
 createdAt: "2023-01-02T04:28:35.508Z"
 updatedAt: "2023-01-15T21:45:31.815Z"
 ---
-# VTEX: The Composable and Complete Commerce Platform
 
-![composable](https://raw.githubusercontent.com/vtexdocs/dev-portal-content/main/images/composable.png)
 
-Our solution delivers a complete set of pre-integrated commerce services that businesses can use to power their ecommerce operation.
+The VTEX platform delivers a complete set of pre-integrated commerce services that businesses can use to power their ecommerce operation.
+
 VTEX provides the building blocks for any kind of ecommerce operation, including:
-Experience management;
-Multiple channel and marketplace management;
-An Order Management System;
-Commerce modules for catalog, checkout, pricing, promotions, account management, and much more.
-On top of that, our add-on solutions amp up sales and take your omnichannel game to the next level by tapping into the latest consumer and market trends, like live shopping, conversational commerce, plus several applications for your store crew and sales team.
+
+- Experience management  
+- Multiple channel and marketplace management  
+- An Order Management System  
+- Commerce modules for catalog, checkout, pricing, promotions, account management, and much more  
+
+
 We understand that every business is unique and no two commerce architectures are exactly alike. That’s why VTEX supports many layers of composability. With our API-first and multi-tenant microservices solution, your tech team can integrate with any sales channel, backend system, or third-party application, extending or swapping our native commerce services for custom and best-of-breed third-party applications that will give your business a competitive edge.
+
+
```

Comment:
```suggestion
```

Reply:
<img width="573" alt="image" src="https://github.com/vtexdocs/dev-portal-content/assets/83723439/9ac23aa3-c12c-44e2-b6df-15e2374e235f">

O heading introduz a imagem, que por sua vez tem a lista de capabilities no diagrama. Sei que não é o ideal de acessibilidade, não ter isso por escrito também, mas the whole point de criar essa doc era pra ter um link compartilhável com essa imagem. Ela vai estar disponível em uma landing page onde falamos sobre composable and complete. Criei a parte escrita do doc só pra ter uma intro à imagem, mas a doc não vai estar listada na navegação, nem na busca.